lib/cog/command/gen_command/base.ex

```diff
@@ -173,8 +178,10 @@ defmodule Cog.Command.GenCommand.Base do
     attr_value(module, :command_name)
   end
 
-  def description(module),
-    do: attr_value(module, :description)
+  for attr <- [:description, :long_description, :examples, :notes, :arguments, :subcommands] do
```

Comment:
Tried to make this change, but there's some weird macro magic getting in the way. Couldn't figure out exactly what was causing it so I'm just going to leave this as is for now.